spring-cloud-kubernetes-examples/kubernetes-circuitbreaker-ribbon-example/name-service/pom.xml

```diff
@@ -23,7 +23,11 @@
 	<parent>
 		<groupId>org.springframework.cloud</groupId>
 		<artifactId>kubernetes-circuitbreaker-ribbon-example</artifactId>
+<<<<<<< HEAD
```

Comment:
You left the merge conflict in here, we should keep 1.1.0.BUILD-SNAPSHOT.  Same with the one below.

Reply:
conflicts resolved